WTWSMS/common/scripted_triggers/ACR_scripted_triggers.txt

```diff
@@ -393,4 +393,21 @@ has_coven_prerequisites = {
 has_sand_snakes_prerequisites = {
 	has_common_devilworship_prerequisites = yes
 	religion_openly_in_egyptian_subgroup_trigger = yes
+}
+
+# Missing true religion triggers
+true_religion_celtic_pagan_trigger = {
+	true_religion = celtic_pagan
+}
+true_religion_celtic_pagan_reformed_trigger = {
+	true_religion = celtic_pagan_reformed
+}
+true_religion_titan_pagan_trigger = {
+	true_religion = titan_pagan
```

Comment:
This religion doesn't exist

Reply:
I did this to clean log output. I can alternatively remove titan_pagan code from customisable localisation.

Reply:
As the religion doesn't exist, that makes more sense yes.

Reply:
Done, removed the code and scripted trigger